src/components/ArchitectHeader.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class ArchitectHeader extends React.Component {
+    render() {
+        return(
+            <p align="center">
+                <a href="//architect.io" target="blank">
+                    <img 
+                        src="https://docs.architect.io/img/logo.svg" 
+                        width="320" 
+                        alt="Architect Logo" 
+                    />
+                </a>
+            </p>
+        ) 
+    }
+}
+
+export default ArchitectHeader;
```

Comment:
Also you should configure your vscode to always add a new line at the end of files.